moncli/entities/column_value/simple.py

```diff
@@ -1,15 +1,75 @@
 from moncli.error import ColumnValueError
 from moncli.entities.column_value.objects import PersonOrTeam
 from moncli import enums
-
+from datetime import datetime, timedelta
 from .base import SimpleNullValue, ComplexNullValue
 from .constants import SIMPLE_NULL_VALUE
 from ...error import ColumnValueError
+from moncli.config import DATE_FORMAT,TIME_FORMAT
+import pytz
 
 class DateValue(ComplexNullValue):
     """A date column value."""
-    pass
+    native_type = datetime
+    allow_casts = (int, str)
+    has_time = False
+
+    def _convert(self,value):
+        try:
+            new_time = datetime.strptime(value['time'], TIME_FORMAT)
+            new_date = datetime.strptime(value['date'], DATE_FORMAT)
+            self.has_time = True
+            new_date = pytz.timezone('UTC').localize(new_date)
+            new_date = new_date + timedelta(hours=new_time.hour, minutes=new_time.minute, seconds=new_time.second)
+            date_value = new_date.astimezone(datetime.now().astimezone().tzinfo) 
+            return date_value
+
+        except KeyError:
+            new_date = datetime.strptime(value['date'], DATE_FORMAT)
+            return new_date
         
+    def _cast(self,value):
+        if isinstance(value,int):
+            try: 
+                new_date_value =  datetime.fromtimestamp(value)
+                self.has_time = True
+                return new_date_value
+
+            except ValueError:
+                raise ColumnValueError(
+                    'invalid_unix_date',
+                     self.id,
+                    'Unable to convert "{}" from UNIX timestamp.'.format(value)
+                )
+
+        if isinstance(value, str):
+            date_value = value.split()
+            try:
+                if len(date_value) == 1:
+                    new_date = datetime.strptime(value['date'], DATE_FORMAT)
```

Comment:
Looks like you forgot to set this back to just _value_, as it is throwing an error because _value_ is a str and does not contain a key-value pair for _date_.